js/webviews.js

```diff
@@ -51,6 +51,16 @@ function onNavigate (tabId, url, isInPlace, isMainFrame, frameProcessId, frameRo
   }
 }
 
+// set default zoom level for tab
+function setTabDefaultZoomLevel (tabId) {
```

Comment:
This isn't working for me. I think the issue is that when it's called from `webviews.add`, the view object hasn't been created in the main process yet, so calling view.setZoomFactor isn't doing anything.

Another thing to consider is how this interacts with user-set zoom levels. For example, right now, if I open Github and zoom it to 150%, Electron will remember that and set the zoom to 150% on my next visit. It seems like this would reset that back to my default zoom level, which seems undesirable.

It looks like webPreferences has a zoomFactor option that can be used to do this instead: https://www.electronjs.org/docs/latest/api/structures/web-preferences. That might work better with custom zoom levels.

(the webPreferences object is created in viewManager.js)